assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -259,6 +259,18 @@ che.infra.kubernetes.namespace=
 # Is used by OpenShift infra as well to specify Project
 che.infra.kubernetes.namespace.default=<username>-che
 
+# add annotations to each user's namespace
+# e.g. { "scheduler.alpha.kubernetes.io/node-selector": "role=spot" }
+che.infra.kubernetes.namespace.annotations_json=NULL
+
+# add labels to each user's namespace
+# e.g. { "eclipse-che": "true" }
+che.infra.kubernetes.namespace.labels_json=NULL
+
+che.infra.kubernetes.namespace.resource_quota_json={"cpu": "16", "memory": "16Gi"}
+che.infra.kubernetes.namespace.limit_range_request_json={"cpu": "0.125", "memory": "200Mi"}
+che.infra.kubernetes.namespace.limit_range_limit_json={"cpu": "2", "memory": "1024Mi"}
```

Comment:
These values should be empty by default; we don't want to suddenly try to create quotas and limit ranges when a user updates Che. It also could fail depending on the cluster and configuration.

This PR also doesn't appear to support removing limit ranges or quotas once they've been created.